app-engine/java/gradle/testSrc/com/google/cloud/tools/intellij/appengine/java/gradle/AppEngineGradleProjectDataServiceTest.java

```diff
@@ -83,4 +103,13 @@ private AppEngineGradleModule createModelAndImportData(boolean hasAppEngineGradl
 
     return appEngineGradleModule;
   }
+
+  @After
+  public void tearDown() {
```

Comment:
Could we move this next to `setUp` method? I think it could make it a bit easier to understand how both setup and tear down work in conjunction.

Reply:
sure, sounds good